x/gov/expected_keepers.go

```diff
@@ -0,0 +1,12 @@
+package gov
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// expected bank keeper
+type BankKeeper interface {
+	GetCoins(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
+
+	// TODO remove once governance doesn't require use of accounts
```

Comment:
Should we open an issue for this work, or is there already one I'm not able to find?

Reply:
open here : https://github.com/cosmos/cosmos-sdk/issues/3628